src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonOutputFormatter.cs

```diff
@@ -25,16 +24,11 @@ public class JsonOutputFormatter : TextOutputFormatter
         // the serializer and invalidate it when the settings change.
         private JsonSerializer _serializer;
 
-        public JsonOutputFormatter()
-            : this(SerializerSettingsProvider.CreateSerializerSettings(), ArrayPool<char>.Shared)
-        {
-        }
-
-        public JsonOutputFormatter(JsonSerializerSettings serializerSettings)
-            : this(serializerSettings, ArrayPool<char>.Shared)
-        {
-        }
```

Comment:
> In this scenario users would have to new up both input and output formatters.

In what scenario would users need a per-controller _input_ formatter? And, despite our penchant for testing corner cases, when would users really need custom settings for a single output formatter?